go/core.rst

```diff
@@ -427,3 +427,57 @@ It references the library(s) being tested with :param:`deps`.
       srcs = ["lib_x_test.go"],
       deps = [":go_default_library"],
   )
+
+go_source
+~~~~~~~~~
+
+This declares a set of source files and related dependancies that can be embedded into on of the
+other rules.
+This is used as a way of easily declaring a common set of sources re-used in multiple rules.
+
+Providers
+^^^^^^^^^
+
+* GoSourceList_
+
+Attributes
+^^^^^^^^^^
+
++----------------------------+-----------------------------+---------------------------------------+
+| **Name**                   | **Type**                    | **Default value**                     |
++----------------------------+-----------------------------+---------------------------------------+
+| :param:`name`              | :type:`string`              | |mandatory|                           |
++----------------------------+-----------------------------+---------------------------------------+
+| A unique name for this rule.                                                                     |
++----------------------------+-----------------------------+---------------------------------------+
+| :param:`srcs`              | :type:`label_list`          | :value:`None`                         |
++----------------------------+-----------------------------+---------------------------------------+
+| The list of Go source files that are compiled to create the package.                             |
+| The following file types are permitted: :value:`.go, .c, .s, .S .h`.                             |
+| The files may contain Go-style `build constraints`_.                                             |
++----------------------------+-----------------------------+---------------------------------------+
+| :param:`deps`              | :type:`label_list`          | :value:`None`                         |
++----------------------------+-----------------------------+---------------------------------------+
+| List of Go libraries this source list imports directly.                                          |
+| These may be go_library rules or compatible rules with the GoLibrary_ provider.                  |
++----------------------------+-----------------------------+---------------------------------------+
+| :param:`embed`             | :type:`label_list`          | :value:`None`                         |
++----------------------------+-----------------------------+---------------------------------------+
+| List of sources to directly embed in this list.                                                  |
+| These may be go_library rules or compatible rules with the GoSourceList_ provider.               |
+| These can provide both :param:`srcs` and param:`deps` to this library.                           |
```

Comment:
Need a colon before param:\`deps\`

Reply:
I did not even read this bit, just cut'n'paste it from one of the 3 other places this broken text is used...
all fixed now.